runtime/compiler/env/j9method.cpp

```diff
@@ -8093,7 +8093,7 @@ TR_J9ByteCodeIlGenerator::runFEMacro(TR::SymbolReference *symRef)
                   switch (argType[0])
                      {
                      case 'L':
-                     case '[':
+                     case '[': //not sure if for 'Q' should we have correspoding "extra_Q" and extraSignature?
```

Comment:
I think it's using `L` and `extra_L` in the `extraSignature` to mark addresses, so I think you can do the same for `Q` for now, but add a TODO.  If `Q` ends up being passed by value ever, it's probably a lot more work.